data/ledger.go

```diff
@@ -76,83 +75,35 @@ type roundSeed struct {
 	elements [2]roundSeedPair
 }
 
-func makeGenesisBlock(proto protocol.ConsensusVersion, genesisBal GenesisBalances, genesisID string, genesisHash crypto.Digest) (bookkeeping.Block, error) {
```

Comment:
why moving into bookkeeping package instead of exporting from data?

Reply:
Because data imports ledger (because it has a type data.Ledger that wraps ledger.Ledger!) .  So ledger can't import data.